proto/proto/geyser.proto

```diff
@@ -157,6 +157,25 @@ message GetHeartbeatIntervalResponse {
   uint64 heartbeat_interval_ms = 1;
 }
 
+message SlotEntryUpdate {
+  // The slot number of the block containing this Entry
+  uint64 slot = 1;
+  // The Entry's index in the block
+  uint64 index = 2;
+  // The number of executed transactions in the Entry
+  // If this number is zero, we can assume its a tick entry
+  uint64 executed_transaction_count = 3;
+}
+
+message TimestampedSlotEntryUpdate {
+  // Time at which the message was generated
+  google.protobuf.Timestamp ts = 1;
```

Comment:
think it'd be better to use timestamp in millis instead of google timestamp, since there is a lot of unwrapping due to the  Option<...> stuff required by prost